Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -35,8 +35,8 @@ function FailBuildOnQualityGateStatus
     {        
         $dashboardUrl = GetTaskContextVariable "MSBuild.SonarQube.ProjectUri"
         
-        Write-Host "##vso[task.logissue type=error]The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
-        Write-Host "##vso[task.complete result=Failed;]"
+        Write-VstsTaskError "The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
```

Comment:
I believe all Output/Error/Warning strings should be localized. We are targeting RC2 for localization. So better to get your strings in now so the translations come back soon. Let me know if you need help with this. It should be pretty easy - all the infrastructure is in place.